ProcessMaker/screen-builder
# Improve Error Handling for Template Application in Screen Builder

## Issue & Reproduction Steps
This PR enhances the error handling mechanism when applying a template to a screen. It ensures that users receive clear error messages instead of the generic 500 server error, improving overall user experience during template application failures.

## Solution
- Updated the error handling logic to return specific error messages by utilizing `error.response.data.error`.

## How to Test
See [Core PR](https://github.com/ProcessMaker/processmaker/pull/7501) for testing steps.

## Related Tickets & Packages
- [FOUR-19320](https://processmaker.atlassian.net/browse/FOUR-19320)
- [Core PR](https://github.com/ProcessMaker/processmaker/pull/7501)
- [Versions Package PR](https://github.com/ProcessMaker/package-versions/pull/98)

ci:processmaker:observation/FOUR-19320
ci:package-versions:observation/FOUR-19320

.


## Code Review Checklist


[FOUR-19320]: https://processmaker.atlassian.net/browse/FOUR-19320?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
